src/app/data-planes/data/index.ts

```diff
@@ -17,7 +19,20 @@ export type DataplaneWarning = {
   }
 }
 
-export type DataplaneNetworking = PartialDataplaneNetworking
+export type DataplaneInbound = PartialDataplaneInbound & {
+  health: {
+    ready: boolean
+  }
+  addressPort: string
+  serviceAddressPort: string
```

Comment:
We need to make these consistent I think? Can we use `socketAddress` to mean both address and port? (and `serviceSocketAddress`), this is what I've used over in `Zones`

Reply:
Ehm, but is that even the same concept?

Reply:
Umm I thought so but maybe not? Don't worry for now.